verification/verification.py

```
@@ -77,7 +83,7 @@ async def refresh_messages(self):
 
                 await message.edit(view=VerificationView(self))
             except discord.errors.NotFound:
-                print(f"Unregistering non existent message with ID: {message['message_id']}")
+                logger.warning(f"Unregistering non existent message with ID: {message['message_id']}")
```

Ruff (0.14.13)

Logging statement uses f-string

(G004)